addon/spaniel-engines/ember-spaniel-engine.js

```diff
@@ -31,5 +35,15 @@ export default {
         });
       });
     }
+  },
+  flushReadQueue() {
```

Comment:
This method isn't really appropriately named, is it? It doesn't really flush the queue, but rather resets the queue. I'd rename it to indicate what it does.

Reply:
In what cases do we have pending work that should be abandoned? 🐟 

The only scenario I can think of would be `destroy` and if so *that* would be the exposed method.

Reply:
The use case we have currently is to handle the early exit - a test failure

```
import {getGlobalEngine} from 'spaniel';
afterTestHook(() => {
  getGlobalEngine().flushAll();
});
```

Reply:
`flush` implies the work is being processed, this does not process the work it merely abandons it. At the very least the naming is incorrect.